draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -973,18 +977,18 @@ most likely believes themselves to be interacting. We'll label this domain the
 "top-level site".
 
 For a document displayed in a top-level browsing context, we can stop here: the
-document's "site for cookies" is the top-level site.
+document's "site for cookies" is top-level site's origin.
```

Comment:
Above (on new line 976), we define "top-level site" as the registrable domain of the URI displayed to the user, which means that calculating its "origin" isn't really a thing you can do (no scheme, no port, etc). I'd suggest that we need to change that term entirely given the new usage. You don't actually want it to be a "site" at all, you now want something like the origin of top-level document. Perhaps doing same-site comparisons against the "top-level origin" would be reasonable?

Reply:
The new core specs have their definition of "origin". See <https://greenbytes.de/tech/webdav/draft-ietf-httpbis-semantics-12.html#origin>.

Reply:
Do HTML's and "URL"'s align? And do they differ from what HTTP says?

Reply:
>  "URL"

Hey there, scare quotes! :) You know this, but I'm referring to https://url.spec.whatwg.org/#origin for the latter, and https://html.spec.whatwg.org/multipage/origin.html#concept-origin for the former. URL's definition of a "URL's origin" depends upon HTML's definition of an opaque origin, and fulfills HTML's definition of a tuple origin.

> And do they differ from what HTTP says?

They do a few things that https://greenbytes.de/tech/webdav/draft-ietf-httpbis-semantics-12.html#origin doesn't, most notably giving guidance for scheme-based rules beyond `http` and `https` (`blob` and `data` are particularly relevant), and expanding upon RFC6465's "globally unique identifier" concept. Cookies need to have consistent rules for both.

Reply:
> Perhaps doing same-site comparisons against the "top-level origin" would be reasonable?

Yep, makes sense. We already use the top-level browsing context's active document's origin in the "site for cookies" algorithm, so we can define a "top-level origin" as the same thing in the prose here.